easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb

```
@@ -0,0 +1,66 @@
+##
+# This file is an EasyBuild recipy as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# https://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Updated to use gompi-2020b toolchain:
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = ['https://github.com/soedinglab/hh-suite/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd67f7f3bf601e48c9c0bc4cf1fbe3b946f787a808bde765e9436a48d27b0964']
+
+builddependencies = [
+    ('CMake', '3.21.1'),
+]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('Python', '3.9.6'),
+]
+
+_binaries = [
+    'hhalign', 'hhalign_mpi', 'hhalign_omp',
+    'hhblits', 'hhblits_mpi', 'hhblits_omp',
+    'hhsearch', 'hhsearch_mpi', 'hhsearch_omp',
+    'hhmake', 'hhfilter', 'hhconsensus',
+]
+
+_scriptfiles = [
+    'hhmakemodel.py', 'hh_reader.py', 'hhsuitedb.py', 'cif2fasta.py',
+]
+
+fix_perl_shebang_for = ['scripts/*pl']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in _binaries] +
+             ['scripts/%s' % y for y in _scriptfiles],
+    'dirs': ['data', 'scripts']
+}
+
```

And could we also include some sanity check commands here for a few (or maybe even all the ones in `_binaries`?) binaries?

---

This seems to work for me:
```suggestion

sanity_check_commands = ['%s -h' % x for x in _binaries]

```

---

will add sanity_check_commands - There are 51 binaries